e2e/run-suite

```diff
@@ -8,6 +8,7 @@ CMD="start"
 PARAMS=""
 SLOWMO=0
 URL=${BASE_URL:-"http://$DEFAULT_HOST:$DEFAULT_PORT"}
+SUITE=${SUITE:-$DEFAULT_SUITE}
```

Comment:
Not sure that introducing another environment variable was the way to go here, could have been another param to `run-suite` but because of my lesser bash skill I chose this path.